### PR TITLE
add profile option

### PR DESCRIPTION
**Pull request by Galithil**
relies on cprofile

### CODE REVIEW

File: scripts/multiqc [hunk 1]
```diff
@@ -333,6 +342,10 @@ plots_flat, plots_interactive, make_pdf, config_file, verbose, quiet, **kwargs):
         pass # No subdirectory variable given
 
 
+    if config.profile:
```
Comment (ewels) [inline scripts/multiqc]: Any reason to start this down here, rather than straight after parsing the command line flags?

File: scripts/multiqc [hunk 2]
```diff
@@ -384,6 +397,14 @@ plots_flat, plots_interactive, make_pdf, config_file, verbose, quiet, **kwargs):
                           this_module, traceback.format_exc()) + ('='*60))
             sys_exit_code = 1
 
+    if config.profile:
+        pr.disable()
```
Comment (ewels) [inline scripts/multiqc]: As above, any reason not to put this right at the end?

**Comment by Galithil** [comment]
Does not provide much value over the use of `python -m cProfile -o <filename>`

**Comment by ewels** [comment]
Thanks for the help anyway! Will write some docs about this now.

**Comment by ewels** [comment]
http://multiqc.info/docs/#profiling-performance

**Pull request closed without merging**